Annotato/Annotato/WebSocket/WebSocketManager.swift

```diff
@@ -45,7 +48,12 @@ class WebSocketManager {
             switch result {
             case .failure(let error):
                 AnnotatoLogger.error(error.localizedDescription)
+                let errorCode = (error as NSError).code
+                if errorCode == 54 || errorCode == 57 || errorCode == 60 {
```

Comment:
Extract these numbers into variables and name them according to what they actually represent.